jobs/tendrl-build-triggers.yml

```diff
@@ -116,3 +116,30 @@
       - node-monitoring
       - performance-monitoring
       - python-gdeploy
+
+
+- project:
+    name: 'develop'
+    project_name: 'develop'
+    target_repo: 'tendrl-test'
+    branch: 'develop'
+    build_commands: 'make snapshot'
+    clustername: build_server
+    copr_config: '~/dahorak_config_copr'
```

Comment:
sure, this is just for testing purposes to be able to verify the process and not to break any existing repo